src/devices/ovrheadset/OVRHeadset.cpp

```diff
@@ -61,6 +63,8 @@ static constexpr unsigned int BUTTON_COUNT = 13;
 #undef check
 #endif
 
```

Comment:
Is this `#ifdef(win) ...` stuff imported from the OVR external library? I think the best place for this stuff is the CMake file, what do you think @drdanz ?stuff